include/layer_shell_v1.h

```diff
@@ -56,6 +56,7 @@ class LayerSurfaceV1
     auto last_height() const -> int { return last_height_; }
 
 private:
+    CheckInterfaceExpected expect_layer_shell_supported;
```

Comment:
I'm not convinced this test belongs in `wlcs::LayerSurfaceV1` and not explicitly in each of the fixtures or tests that use it.

In any case, it doesn't need to be a member variable - it can equally be local to the constructor.

Reply:
This has been resolved